Makefile

```diff
@@ -8,6 +8,7 @@ generate:
 	find pkg/ -name "*generated*.go" -exec rm {} -f \;
 	./hack/build-go.sh generate ${WHAT}
 	goimports -w -local kubevirt.io cmd/ pkg/ tests/
+	./hack/bootstrap-ginkgo.sh test ${WHAT}
```

Comment:
Make generate is being used for formatting and documentation. I don't personally like the idea of running the test suite at that point. We already have the `make test` and `make functest` targets. Isn't that enough?

EDIT: Or am I misreading this line because of the `test` argument, which actually appears to be unused in `hack/bootstrap-ginkgo.sh`?